src/main/java/hudson/plugins/git/GitSCM.java

```diff
@@ -1388,7 +1398,6 @@ public void buildEnvironment(Run<?, ?> build, java.util.Map<String, String> env)
             }
         }
 
-
```

Comment:
Please restore this blank line.  I realize it is an annoyance to have inconsistent formatting in so many places, but the large number of pending pull requests means that smaller changes are less likely to cause one of the pending pull requests to have a whitespace related conflict.